python/modules/IcePy/msbuild/icepy.vcxproj.filters

```diff
@@ -84,19 +84,19 @@
     <ClCompile Include="..\..\..\..\cpp\src\Slice\Preprocessor.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\..\cpp\src\Slice\Python.cpp">
+    <ClCompile Include="..\..\..\..\cpp\src\Slice\Scanner.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\..\cpp\src\Slice\PythonUtil.cpp">
+    <ClCompile Include="..\..\..\..\cpp\src\Slice\SliceUtil.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\..\cpp\src\Slice\Scanner.cpp">
+    <ClCompile Include="..\..\..\..\cpp\src\Slice\StringLiteralUtil.cpp" />
+    <ClCompile Include="..\..\..\..\cpp\src\slice2py\Python.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\..\cpp\src\Slice\SliceUtil.cpp">
+    <ClCompile Include="..\..\..\..\cpp\src\slice2py\PythonUtil.cpp">
```

Comment:
Is this change correct?

Reply:
Seems correct to me, the is a bit confusing here, but PythonUtil comes from slice2py and SliceUtil from cpp\src\Slice

Reply:
`Python.cpp` -> `Scanner.cpp`?

Reply:
Or am I just reading it wrong?

Reply:
Yeah, the diff is non the clearest, but in the actual file, everything seems correct.